week 10 sharing

furkansindar opened the pull request:
I finish week 10 sharing

furkansindar committed:
week 10 sharing

I finish week 10 sharing

tsteelematc commented:
Do you plan to hook this up to the service and get real planets?

It would be similar to Uger's component, which I believe I demoed/discussed in class...

<img width="2217" height="678" alt="image" src="https://github.com/user-attachments/assets/fd547a4d-041b-4af3-9885-8a92205f637b" />


And some code I wrote that does this...

https://github.com/2025-fall-js-frameworks/angular-components/blob/tsteele/uger-nudge/src/app/ugur-my-component/ugur-my-component.ts

furkansindar commented:
For now I want to keep my own local planets. Later I can switch to the Star Wars planets from the list.

tsteelematc committed:
This uses your array to filter the planets from the service...

Then map them with the descriptions...

Can you explain why Endor is no longer showing up?

tsteelematc commented on src/styles.css:
No changes expected in this file

tsteelematc commented on src/custom-theme.scss:
No changes expected in this file

furkansindar committed:
week 10 sharing

backing up my changes

furkansindar committed:
Merge pull request #43 from 2025-fall-js-frameworks/tsteele/fsindar-week-10-sharing-nudge

This uses your array to filter the planets from the service...

furkansindar committed:
finding the issue

finding the issue of my codes
